packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -70,131 +70,172 @@ export type ChipItem = {
   hidden?: boolean;
 } & SingleSelectChipItem;
 
+export type InternalProps = {
+  chipGroupClassNames: string
+};
+
 export type CommonProps = {
   label?: string;
   ariaLabel?: string;
-  type?: ChipGroupType;
   chipStyle?: ChipStyleType;
-  // only required when type = rail
-  leadingNudgerLabel?: string;
-  trailingNudgerLabel?: string;
+  chips: ChipItem[];
+  ariaMultiselectable?: boolean;
 };
 
-export type ChipGroupProps = {
-  chips: ChipItem[];
+export type RailChipGroupProps = {
   stickyChip?: ChipItem;
-  ariaMultiselectable?: boolean;
+  leadingNudgerLabel?: string;
+  trailingNudgerLabel?: string;
```

Comment:
These two labels should now be required since they were previously required for rail but not needed for wrap. Now this type is rail only they can be required and the default of `""` removed